addons/im_livechat/models/res_users.py

```diff
@@ -12,15 +12,32 @@ class ResUsers(models.Model):
 
     livechat_username = fields.Char(string='Livechat Username', compute='_compute_livechat_username', inverse='_inverse_livechat_username', store=False)
     livechat_lang_ids = fields.Many2many('res.lang', string='Livechat Languages', compute='_compute_livechat_lang_ids', inverse='_inverse_livechat_lang_ids', store=False)
+    livechat_expertise_ids = fields.Many2many(
+        "im_livechat.expertise",
+        string="Live Chat Expertise",
+        compute="_compute_livechat_expertise_ids",
+        inverse="_inverse_livechat_expertise_ids",
+        store=False,
+        help="When forwarding live chat conversations, the chatbot will prioritize users with matching expertise.",
```

Comment:
ditto for the chat bot mention.

Reply:
Not changing in this PR as this is the only flow where it is used, which helps understanding the purpose when setting it up.
We can adapt the help if we have other use case in future PR.